libcontainer/intelrdt/intelrdt.go

```diff
@@ -167,10 +169,16 @@ type IntelRdtManager struct {
 	Config *configs.Config
 	Id     string
 	Path   string
+	Type   string
 }
 
 const (
-	IntelRdtTasks = "tasks"
+	IntelRdtTasks         = "tasks"
+	MonGroupDirectoryName = "mon_groups"
+	// Monitoring group.
+	MON = "MON"
```

Comment:
can you make these more descriptive names.  looking through the code where these are used, it's hard to tell what they are switching on.

Reply:
Is it now more descriptive?